### PR TITLE
DL-3004 SBT Update

### PR DESCRIPTION
**Pull request by Sir-Iyke**
DL-3004 SBT Update

**New feature** 

Upgraded SBT and Scala dependencies

## Checklist

Iyke
 - [X]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [X]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [X]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [X]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [X]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -61,4 +63,4 @@ trait MicroService {
       )
     )
     .enablePlugins(SbtDistributablesPlugin, SbtAutoBuildPlugin, SbtGitVersioning)
-}
+
```
Comment (paulanderson1234) [inline build.sbt:66]: Need to add entry for  .disablePlugins(JUnitXmlReportPlugin) to sbt file otherwise the build will fail

**Comment by paulanderson1234** [comment]
Also need additional config changes to service-manager-config and build-jobs to support this PR

File: build.sbt (hunk 1 above)
Comment (Sir-Iyke) [reply build.sbt:68]: Done

**Comment by Sir-Iyke** [comment]
Raised PR for service manager config

**Comment by Sir-Iyke** [comment]
https://github.com/hmrc/service-manager-config/pull/2142

**Pull request merged**